src/Microsoft.Health.Fhir.CosmosDb.Initialization.UnitTests/CosmosDbInitializationTests.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.StoredProcedures;
+using Xunit;
+
+namespace CosmosDb.Initialization.UnitTests
```

Comment:
The namespace should have the prefix "Microsoft.Health.Fhir.".

Reply:
done